EME FAQ

FilipeFcp opened the pull request:
Hi all.
I added some FAQ questions for the EME solver. Let me know if you have any suggestions.

Thanks!

FilipeFcp committed:
EME FAQ

e-g-melo reviewed (approved):
Thanks @FilipeFcp! The questions are very helpful.

I only found this typo: "How can I simulate bent waveugides with EME?" -> "How can I simulate bent waveguides with EME?"

FilipeFcp committed:
correcting typos

FilipeFcp commented:
Thanks @e-g-melo.

caseyflex reviewed (approved):
Thanks Filipe, this will be very helpful! It is a nice and clear introduction to our solver.

- Is it recommended to use double-precision when accurate bend loss is important?
- Should we mention EMEPeriodicitySweep in sweeps?

FilipeFcp committed:
periodic sweep

FilipeFcp committed:
double precision

FilipeFcp commented:
Hi @caseyflex 
Regarding double precision, if I remember correctly, the main issue is when the losses are very small, so the imaginary part of the refractive index can’t be represented using single precision. I added a note on that.

I also added the periodic sweep and a dedicated FAQ to it

caseyflex commented:
> Hi @caseyflex Regarding double precision, if I remember correctly, the main issue is when the losses are very small, so the imaginary part of the refractive index can’t be represented using single precision. I added a note on that.
> 
> I also added the periodic sweep and a dedicated FAQ to it

thanks! looks good
